.eslintrc

```diff
@@ -1,7 +1,7 @@
 {
     "root": true,
     "parserOptions": {
-        "ecmaVersion": 6,
+        "ecmaVersion": 2019,

```

Comment:
Is this change necessary to fix 401's?

Reply:
It has been pulled out to a seperate pr, merged to master and rebased on.

Its because I wanted to use object spread...